simapp/doc.go

```diff
@@ -0,0 +1,93 @@
+/*
+Package simapp implements a full fledged Cosmos SDK application used for executing
+simulation test suites.
+
+Simulation App
+
+The SimApp defines an application used for running extensive simulation
+testing suites. It contains all core modules, including governance, staking,
+slashing, and distribution.
+
+Simulation is executed with various inputs including the number of blocks to
+simulate, the block size, whether the app should commit or not, the invariant
+checking period, and a seed which is used as a source of pseudo-randomness.
+
+In addition to the various inputs, simulation runs mainly in three modes:
+
+1.) Completely random where the initial state, module parameters and simulation
```

Comment:
Bullet point lists are not supported, I don't think it's a great deal though.
I'd anyway remove either `.` or `)`, having both feels redundant. Good job :+1:

Reply:
Ok, yeah I'll go with `)`.